addons/mail/static/src/core/web/chatter.js

```diff
@@ -194,11 +194,15 @@ export class Chatter extends Component {
                 } else {
                     this.state.showAttachmentLoading = false;
                     this.state.isAttachmentBoxOpened =
-                        this.props.isAttachmentBoxVisibleInitially && this.attachments.length > 0;
+                        this.state.isAttachmentBoxOpened && this.attachments.length > 0;
                 }
                 return () => browser.clearTimeout(this.loadingAttachmentTimeout);
             },
-            () => [this.state.thread, this.state.thread?.isLoadingAttachments]
+            () => [
+                this.state.thread,
+                this.state.thread?.isLoadingAttachments,
+                this.attachments.length,
+            ]
```

Comment:
It looks like the dependencies of this `useEffect` were made specifically to handle the `showAttachmentLoading`, I'm not really sure why `isAttachmentBoxOpened` is handled there.

Also why do we have `isLoadingAttachments` and a different field `status` `loading`?